TerminalDocs/customize-settings/global-settings.md

```diff
@@ -176,21 +176,21 @@ When set to `true`, this enables the launch of Windows Terminal at startup. Sett
 
 **Default value:** `false`
 
-### Launch size
+### Launch mode
 
-This defines whether the terminal will launch as maximized, full screen, or in a window.
+This defines whether the terminal will launch as maximized, full screen, or in a window. Setting this to `focus` is equivalent to launching the terminal in the `default` mode, but with the focus mode enabled. Similar, setting this to `focusMaximized` will result in launching the terminal in a maximized window w ith the focus mode enabled.
 
 **Property name:** `launchMode`
 
 **Necessity:** Optional
 
-**Accepts:** `"default"`, `"maximized"`, `"fullscreen"`
+**Accepts:** `"default"`, `"maximized"`, `"fullscreen"`, `"focus"`, `"maximizedFocus"`
 
 **Default value:** `"default"`
```

Comment:
Same here, I'd add a


```md
> [!IMPORTANT]
> The `"focus"` and `"maximizedFocus"` modes are only available in [Windows Terminal Preview](https://aka.ms/terminal-preview/), version 1.5+.
```

blob to make sure users know which version these properties were introduced in.